simmechanics/data/icub3/ICUB_3_all_options_gazebo.yaml

```diff
@@ -304,6 +304,8 @@ sensors:
   #          <yarpConfigurationFile>model://iCub/conf_icub3/gazebo_icub_inertial.ini</yarpConfigurationFile>
   #      </plugin>
 
+@ASSIGNED_COLLISION_GEOMETRIES@

```

Comment:
Sorry I am late 😓 I think that this file should be renamed in `yaml.in` since it is no more usable without cmake, what do you think about it?

Reply:
I will open a new PR for that. Thanks for noticing.

Reply:
Thanks @Nicogene , I had missed this.